pkg/security/serializers/serializers_base.go

```diff
@@ -344,6 +352,13 @@ func newMatchedRulesSerializer(r *model.MatchedRule) MatchedRuleSerializer {
 	return mrs
 }
 
+// nolint: deadcode, unused
+func newFailedDNSEventSerializer(d *model.FailedDNSEvent) *FailedDNSEventSerializer {
+	return &FailedDNSEventSerializer{
+		Payload: base64.StdEncoding.EncodeToString(d.Payload),
+	}
+}
```

Comment:
More or less the same comment here: the FailedDNSEvent type from custom_events.go has already its marshaller ToJSON method